opentech/apply/projects/views.py

```diff
@@ -0,0 +1,7 @@
+from django.views.generic import DetailView
+
+from .models import Project
+
+
+class ProjectDetailView(DetailView):
```

Comment:
We'll need to `login_required` this view at some point. Can you make this:
`@method_decorator(staff_required, name='dispatch')`
and we'll handle the user stuff at a later date?

Reply:
Fixed in 9ee19678